yb-voyager/cmd/importData.go

```diff
@@ -1643,6 +1647,8 @@ func cleanMSRForImportDataStartClean() error {
 	if !startClean {
 		log.Infof("skipping cleaning migration status record for import data command start clean")
 		return nil
+	} else if !isPrimaryKeyConflictIgnoreModeValid() {
```

Comment:
the function name is generic (`cleanMSRForImportDataStartClean` ), so it seems weird that we skip if `!isPrimaryKeyConflictIgnoreModeValid` . 

I'd recommend:
- either change the function name to make it specific(it doesn't do anything other than resetting OnPrimaryKeyConflictAction, so that makes sense)
- keep the function name as-is, but only run 1662-1664 if `isPrimaryKeyConflictIgnoreModeValid`

Reply:
I actually removed the check from `cleanMSRForImportDataStartClean`.
I don't think this required to be checked, we can erase irrespectively.